common/system/hardware_manager.c

```diff
@@ -958,7 +959,7 @@ void hm_pyroSetPwm(int pyroId, uint32_t duration, uint32_t frequency,
 #endif  // HAS_DEV(PYRO_DIGITAL)
 }
 
-void hm_pyroUpdate() {
+void hm_pyroUpdate(void *pUserData) {
```

Comment:
Out of scope here, but probably worth refactoring some of our raw callbacks if it means that we're having to add unused parameters to avoid warnings.

Reply:
Tracked in #268